pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -1777,6 +1795,7 @@ func doAppNetworkConfigModify(ctx *zedrouterContext, key string,
 		config.UUIDandVersion, config.DisplayName)
 
 	if !doAppNetworkSanityCheckForModify(ctx, config, status) {
+		publishAppNetworkStatus(ctx, status)
```

Comment:
Is this missing a PendingModify = false, or is the caller doing that?

Reply:
the caller is doing this.

Reply:
OK